app/services/address_validation_service.rb

```diff
@@ -18,4 +18,11 @@ def invalid?
     !@validation.verifications.delivery.success ||
       @validation.street1.include?('PO BOX')
   end
+
+  def error
+    return nil unless @validation.present? &&
+                      @validation.verifications.delivery.errors.present?
+    return 'PO BOX' if @validation.street1.include?('PO BOX')
```

Comment:
You return 'PO BOX' if it includes that, but is there a custom message for indicating this or is the ERROR.ADDRESS_VALIDATION always the default error message?